pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -56,10 +56,12 @@ def expected_customer_spend(
         mean_transaction_value, frequency = to_xarray(
             customer_id, mean_transaction_value, frequency
         )
-        assert self.idata is not None, "Model must be fitted first"
-        p = self.idata.posterior["p"]
-        q = self.idata.posterior["q"]
-        v = self.idata.posterior["v"]
+        if (posterior := self.fit_result) is None:
+            raise ValueError("Model must be fitted first")
```

Comment:
This doesn't work? self.fit_result will already raise if needed

```suggestion
        posterior = self.fit_result
```